uint8 motor_direction
uint8 motor_speed
bool launcher_heater
